pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/omstillingsstoenad/informasjon/OmstillingsstoenadInformasjonDoedsfall.kt

```diff
@@ -0,0 +1,258 @@
+
+                title2 {
+                    text(
+                        Bokmal to "Andre ytelser du kan ha rett til som gjenlevende ",
+                        Nynorsk to "Andre ytingar du kan ha rett til som attlevande ",
+                        English to "",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Du kan også ha rett til stønad til barnetilsyn, tilleggsstønad og stønad til skolepenger. Forsørger du barn under 18 år, kan du ha rett til utvidet barnetrygd. ",
+                        Nynorsk to "Dersom avdøde hadde ei privat eller offentleg pensjonsordning og du har spørsmål om dette, kan du kontakte arbeidsgivaren til den avdøde. Du kan også ta kontakt med pensjonsordninga eller forsikringsselskapet. ",
+                        English to "",
+                    )
+                }
+
+                title2 {
+                    text(
+                        Bokmal to "Andre pensjonsordninger",
+                        Nynorsk to "Andre pensjonsordningar",
+                        English to "Other pension schemes",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Dersom avdøde hadde en privat eller offentlig pensjonsordning og du har spørsmål om dette, kan du kontakte avdødes arbeidsgiver. Du kan også ta kontakt med pensjonsordningen eller forsikringsselskapet. ",
+                        Nynorsk to "Dersom avdøde hadde ei privat eller offentleg pensjonsordning og du har spørsmål om dette, kan du kontakte arbeidsgivaren til den avdøde. Du kan også ta kontakt med pensjonsordninga eller forsikringsselskapet. ",
+                        English to "If the deceased was a member of a private or public pension scheme and you have questions about this, you can contact the deceased's employer. You can also contact the pension scheme or insurance company. ",
+                    )
+                }
+
+                title2 {
+                    text(
+                        Bokmal to "Har du spørsmål?",
+                        Nynorsk to "Har du spørsmål?",
+                        English to "Do you have any questions?",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Du finner mer informasjon på nav.no/omstillingsstønad. På nav.no/kontakt kan du chatte eller skrive til oss.",
+                        Nynorsk to "Du finn meir informasjon på nav.no/omstillingsstønad. Du kan chatte med oss eller skrive til oss på nav.no/kontakt.",
+                        English to "You can find more information at nav.no/omstillingsstønad.",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Hvis du ikke finner svar på nav.no, kan du ringe oss på telefon + 47 55 55 33 34, hverdager kl. 09.00 - 15.00.",
```

Comment:
Er det noen grunn til at vi skriver fra 9 til 15 på tre ulike måter? 
nb: `09.00 - 15.00.`
nn: `09.00 til 15.00.`
en: `09:00 to 15:00.`

Reply:
Sikkert lokale forskjeller

Reply:
Fra spøk til alvor så er vel det oversetterens valg?